MultiServer.py

```diff
@@ -1050,17 +1050,19 @@ def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bo
         if picks[0][1] == 100:
             return picks[0][0], True, "Perfect Match"
         elif picks[0][1] < 75:
-            return picks[0][0], False, f"Didn't find something that closely matches, " \
-                                       f"did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Didn't find something that closely matches '{input_text}', " \
+                                       f"did you mean '{picks[0][0]}'? ({picks[0][1]}% sure)"
         elif dif > 5:
             return picks[0][0], True, "Close Match"
         else:
-            return picks[0][0], False, f"Too many close matches, did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Too many close matches to '{input_text}', " \
```

Comment:
Not a native speaker, but "to" sounds wrong to me here..
```suggestion
            return picks[0][0], False, f"Too many close matches for '{input_text}', " \
```

Reply:
I think both are correct, but "to" is usually used in the context of something being similar to something, like it is in this case.

Reply:
I think I prefer "for", personally, but I do believe both are valid and "good enough"

Reply:
Alright, sounds good to me. :+1: